src/fabric/src/fabric2_util.erl

```diff
@@ -40,7 +40,9 @@
     encode_all_doc_key/1,
 
     pmap/2,
-    pmap/3
+    pmap/3,
+    iso8601_timestamp/0,
+    do_recovery/0
```

Comment:
Super minor nit but lest move these above `pmap/N` into their own section. Don't forget to move the implementations as well.